zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -615,6 +615,12 @@ def _getClangCMakeBuildFactory(
                                description='recreating sandbox',
                                workdir='test',
                                env=env))
+        f.addStep(ShellCommand(name='install lnt dependencies',
+                               command=[python, '-m', 'pip', 'install', 'setuptools'],
```

Comment:
Given that "setup lit" uses the sandbox's Python:
```
/home/tcwg-buildbot/worker/clang-aarch64-sve-vla/test/sandbox/bin/python /home/tcwg-buildbot/worker/clang-aarch64-sve-vla/test/lnt/setup.py develop
```
I think we are installing setuptools into the sandbox itself here.

Which is what we want to do, so we don't conflict with existing installs.

It's just not very clear from `workdir=` whether it will activate the venv, or magically make that the PATH. I guess it puts that on the PATH somehow.

Anyway, whatever it does, it works.

Reply:
The env is activated by calling python from $env/bin directory.  This is arrange through "python", which is a variable, not a string.